log/log.go

```diff
@@ -0,0 +1,78 @@
+/**
+*  @file
+*  @copyright defined in go-seele/LICENSE
+ */
+package log
+
+import (
+    "fmt"
+    "github.com/sirupsen/logrus"
+    "os"
+    "os/user"
+    "path/filepath"
+)
+
+var log *logrus.Logger
+
+// PanicLevel level, highest level of severity. Logs and then calls panic with the
```

Comment:
nice to add a space between Panic and Level:
PanicLevel  --> Panic Level

to meet golint format requirement.

Reply:
sure